### PR TITLE
Advertise FT better in README

### PR DESCRIPTION
**Pull request by efb4f5ff-1298-471a-8973-3d47447115dc**
# Advertise FT better in README

## Pull Request Type

- [ ] Bugfix
- [ ] Feature Implementation
- [x] Documentation
- [ ] Other

## Description

Notable things that I've done:

- Ordered the headings in a more logical structure for readers
- removed some sentences that were redundant 
- removed browser extension disclaimer because the link didn't provide the adequate info anymore and couldn't find a good replacement

## Screenshots 

I think its better to go to https://github.com/efb4f5ff-1298-471a-8973-3d47447115dc/FreeTube/blob/marketing/README.md

## Additional context

Need your help/opinion on:

- Did i miss some features that we should definitely list?
- Should we update our pictures to show more of our features?
- i've made the nighly builds header smaller because it is a subset of the official downloads but the header is now the same size as plain text. Should i revert this change?
- Do we want to add badges/widgets in the same way https://github.com/privacyguides/privacyguides.org did above their about header?
- Do we want to include the following statement in the extension section?
> In order for the extension to work properly, you must be using an installable, non-portable version of FreeTube. This is because non-portable builds do not setup the required protocol needed for your operating system to redirect the URLs over to FreeTube.
The following builds are supported:
    .exe
    .dmg
    .deb
    .rpm
    Flatpak

### CODE REVIEW

**Review by absidue — APPROVED** [review]

**Review by PikachuEXE — DISMISSED** [review]

**Review by absidue — APPROVED** [review]

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Why nobody reading the Additional context section :(

**Comment by PikachuEXE** [comment]
> Did i miss some features that we should definitely list?

Having a Pikachu as developer /s

> Should we update our pictures to show more of our features?

No, at most link to a page on docs and update docs website to display images (otherwise this README might become very long)

> i've made the nighly builds header smaller because it is a subset of the official downloads but the header is now the same size as plain text. Should i revert this change?

Fine for me ._.

> Do we want to add badges/widgets in the same way https://github.com/privacyguides/privacyguides.org did above their about header?

Change welcome but separate PR preferred

> Do we want to include the following statement in the extension section?
In order for the extension to work properly, you must be using an installable, non-portable version of FreeTube.

Is it repeatedly asked? If so include it

**Comment by ChunkyProgrammer** [comment]
Not sure about including the screenshot one. It'd definitely a nice to have but it's pretty easy to take a screenshot outside of thevapp (on windows, you just have to do Win + Shift + S and you can select an area to screenshot). Iirc the screenshot only takes a screenshot of the video so maybe that should be included in the line if you still want to keep it? 


I also think you should include that part in the extension section that says not all builds are supported

File: README.md [hunk 1]
```diff
@@ -6,63 +6,71 @@ FreeTube is an open source desktop YouTube player built with privacy in mind.
...
-## Screenshots
-<img src="https://i.imgur.com/zFgZUUV.png" width=300> <img src="https://i.imgur.com/9evYHgN.png" width=300> <img src="https://i.imgur.com/yT2UzPa.png" width=300> <img src="https://i.imgur.com/47zIEt4.png" width=300> <img src="https://i.imgur.com/hFB2fKC.png" width=300>
-
 ## Features
 * Watch videos without ads
 * Use YouTube without Google tracking you using cookies and JavaScript
 * Two extractor APIs to choose from (Built in or Invidious)
 * Subscribe to channels without an account
-* Local subscriptions, history, and saved videos
+* Connect to an externally setup tor proxy
```
[the remark this answers is not in the document]
Comment (ChunkyProgrammer) [reply README.md:33]: Maybe ` Connect to an externally setup proxy such as Tor`

[the remark this answers is not in the document]
Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply README.md:33]: sound good to me

File: README.md [hunk 2]
```diff
@@ -6,63 +6,85 @@ FreeTube is an open source desktop YouTube player built with privacy in mind.
...
+
+* .dmg
+
+* .exe
+
+* Flatpak
+
+* Portable (Windows only)
+
+* .rpm
```
Comment (PikachuEXE) [inline README.md:72]: Why are these so separated?

Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply README.md:72]: they need to be removed stop using ur thunderbolt so fast :'(

**Review by absidue — APPROVED** [review]

**Review by ChunkyProgrammer — APPROVED** [review]

**Review by PikachuEXE — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (PikachuEXE) [reply README.md]: No
image

**Pull request merged**